ast/Helpers.cc

```diff
@@ -90,6 +89,10 @@ bool BehaviorHelpers::checkEmptyDeep(const ExpressionPtr &expr) {
                      checkEmptyDeep(seq.expr);
         },
 
+        [&](const ast::RuntimeMethodDefinition &) { result = true; },
+
+        [&](const ast::KeepForIDE &) { result = true; },
+
```

Comment:
We don't have great tools to detect cases where we need to start handling new pieces of syntax... It's the only thing that worries me when adding new tree nodes.